packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -221,7 +221,7 @@ class VerboseLogger extends Logger {
     stopwatch.reset();
 
     String prefix;
-    const int prefixWidth = 12;
+    const int prefixWidth = 8;
```

Comment:
I don't mind if we make this narrow, but I see things in the range of hours quite often, because I often have multi-hour sessions and sometimes (e.g. when I go to lunch) come back and the next thing has a multi-hour time delta, and it looks ugly. So if we stick to a narrow time, can we do something like change the units or something?